qa/evil-tests/src/test/java/org/elasticsearch/tribe/TribeUnitTests.java

```diff
@@ -59,8 +60,24 @@ public static void createTribes() {
             .put("node.mode", NODE_MODE)
             .put("path.home", createTempDir()).build();
 
-        tribe1 = NodeBuilder.nodeBuilder().settings(Settings.builder().put(baseSettings).put("cluster.name", "tribe1").put("node.name", "tribe1_node")).node();
-        tribe2 = NodeBuilder.nodeBuilder().settings(Settings.builder().put(baseSettings).put("cluster.name", "tribe2").put("node.name", "tribe2_node")).node();
+        tribe1 = NodeBuilder
+            .nodeBuilder()
+            .settings(
+                Settings.builder()
+                    .put(baseSettings)
+                    .put("cluster.name", "tribe1")
+                    .put("node.name", "tribe1_node")
+                    .put(DiscoveryService.SETTING_DISCOVERY_SEED, random().nextLong())
```

Comment:
> but, it seems we look for an hexadecimal number? (which is OK because normal long are also parseable as hex, but you know, pedantic)

When parsing the system property "tests.seed" we look for a hexadecimal number (because this is the format that randomizedtesting provides seeds in). When parsing a setting (e.g., DiscoveryService#SETTING_DISCOVERY_SEED", we look for a decimal number.